### PR TITLE
[amqphandler] Don't truncate debug logs

### PR DESCRIPTION
**Pull request by mlohvynenko**

### CODE REVIEW

File: amqphandler/amqphandler.go [hunk 1]
```diff
@@ -60,6 +60,8 @@ const (
 type AmqpHandler struct { //nolint:stylecheck
 	sync.Mutex
 
+	disableLogsTrim bool
```
Comment (al1img) [inline amqphandler/amqphandler.go:63]: Don't see why we need to have it configurable. Let's just remove log trimming at all.

File: config/config.go [hunk 2]
```diff
@@ -110,6 +110,7 @@ type Config struct {
 	Migration             Migration         `json:"migration"`
 	SMController          SMController      `json:"smController"`
 	UMController          UMController      `json:"umController"`
+	DisableLogsTrim       bool              `json:"disableLogsTrim"`
```
Comment (al1img) [inline config/config.go:113]: Should name `TrimAMQPDataLogs" and should be false by default.

File: amqphandler/amqphandler.go [hunk 3]
```diff
@@ -146,12 +148,13 @@ var (
  **********************************************************************************************************************/
 
 // New creates new amqp object.
-func New() (*AmqpHandler, error) {
+func New(disableLogsTrim bool) (*AmqpHandler, error) {
```
Comment (al1img) [inline amqphandler/amqphandler.go:151]: trimDataLogs

File: amqphandler/amqphandler.go [hunk 4]
```diff
@@ -60,6 +60,8 @@ const (
 type AmqpHandler struct { //nolint:stylecheck
```
Comment (mykola-kobets-epam) [inline amqphandler/amqphandler.go:60]: May be better make maxLenLogMessage configurable?

Comment (mlohvynenko) [reply amqphandler/amqphandler.go:60]: As per @al1img  suggestion I will rename it to `TrimAMQPDataLogs`
I considered this option prior, but this naming might be confusing: only IAM AMQP message is truncated. Users might treat `maxLenLogMessage` as an option to limit all logs size for CM

File: amqphandler/amqphandler.go (hunk 1 above)
Comment (mlohvynenko) [reply amqphandler/amqphandler.go:63]: JIRA task: https://jiraeu.epam.com/browse/EPMPAOS-5351
Title: [AOS Core] Add the ability not to truncate the unit's messages to the cloud.

So the task is to add an ability to not truncate logs

File: config/config.go (hunk 2 above)
Comment (mlohvynenko) [reply config/config.go:113]: Applied suggestion, but made default value true

File: amqphandler/amqphandler.go (hunk 1 above)
Comment (mlohvynenko) [reply amqphandler/amqphandler.go]: Agreed to not truncate logs at all.
Updated PR.

**Review by al1img — APPROVED** [review]

**Review by mykola-kobets-epam — APPROVED** [review]

**Pull request merged**